website/core/Head.js

```diff
@@ -19,6 +19,16 @@ const Head = React.createClass({
         <meta property="og:image" content="https://facebook.github.io/jest/img/opengraph.png" />
         <meta property="og:description" content={this.props.description} />
 
+
+        <link rel="apple-touch-icon" sizes="180x180" href="/jest/img/favicon/apple-touch-icon.png" />
+        <link rel="icon" type="image/png" sizes="32x32" href="/jest/img/favicon/favicon-32x32.png" />
+        <link rel="icon" type="image/png" sizes="16x16" href="/jest/img/favicon/favicon-16x16.png" />
+        <link rel="manifest" href="/jest/img/favicon/manifest.json" />
+        <link rel="mask-icon" href="/jest/img/favicon/safari-pinned-tab.svg" color="#5bbad5" />
```

Comment:
No, that is not how it supposed to look. I haven't check on safari, sorry.
Thanks for the feedback.
I will make the wanted modification.